.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: node_js
 node_js:
-  - 6
-  - 8
   - 10
-  - stable
+  - 12
 
 before_install:
   - npm config set loglevel warn
 
-before_script:
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+services:
+  - xvfb
+
+script:
+  - commitlint-travis
```

Comment:
Hm.. this means `npm test` is no longer executed by the CI build?

Reply:
sorry didn't realize that was triggered by default. PTAL